### DISCUSSION TITLE
日志总是相差8小时

### DISCUSSION THREAD

**Discussion by wcbspp** [messense/aliyundrive-webdav#801] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/125283858/230707436-164269fe-c23d-4755-b162-31847a1370b5.png)
 Z 结尾的时间是好像是 UTC 时间 ，怎么才能切换回东八区 看着好难受~ 其他插件日志正常。

**Comment by messense** [comment, marked as answer]
先忍忍，fixed in https://github.com/messense/aliyundrive-webdav/commit/5079202745a7e1ba3c7e6a57ee8124d9d147dfbd.

**Reply by wcbspp** [reply]
🫡> 先忍忍，fixed in https://github.com/messense/aliyundrive-webdav/commit/5079202745a7e1ba3c7e6a57ee8124d9d147dfbd.
